### ISSUE TITLE
Adzes and softwood/hardwood confusion

### ISSUE THREAD

**Issue by jeremyshannon** [jeremyshannon/Exile#97]
Adzes are an important stone-age tool, but new players can be confused if they try to cut a maraka tree and the adze does nothing.
 Add one or both of A) a tooltip that explains the adze is used to cut softwood trees B) differing sound effects for a hardwood vs softwood tree

**Comment by DokimiCU** [comment]
Agree. Sound effects are always a good idea, they give that little bit of extra info in a non-intrusive way/

Project history:
The reason for preventing adze from cutting hard wood was because historically people typically wouldn't fell a whole tree with one. They'd use burning or ring barking.
It was the same issue with pick ax vs granite. Sure you could dig/cut that if you really wanted, but it would take so long no one would bother. My design choice was just to make it impossible, which does create this confusion unfortunately.

(For hard stones adding explosives or fire-cracking was something in the back of my mind. Was always too much trouble though).
